File: library/src/main/java/com/senierr/adapter/support/bean/LoadMoreBean.java
package com.senierr.adapter.support.bean;

/**
 * 加载更多实体类
 *
 * @author zhouchunjie
 * @date 2017/11/3
 */
public class LoadMoreBean {

    public final static int STATUS_LOADING = 101;
    public final static int STATUS_COMPLETED = 102;
    public final static int STATUS_NO_MORE = 103;
    public final static int STATUS_FAILURE = 104;

    private int loadState;

    public int getLoadState() {
        return loadState;
    }

    public void setLoadState(int loadState) {
        this.loadState = loadState;
    }
}

File: multitype/src/main/java/com/senierr/adapter/support/bean/LoadMoreBean.kt
package com.senierr.adapter.support.bean

/**
 * 加载更多实体类
 *
 * @author zhouchunjie
 * @date 2017/11/3
 */
data class LoadMoreBean(var loadState: Int = STATUS_NO_MORE) {
    companion object {
        const val STATUS_LOADING = 101
        const val STATUS_COMPLETED = 102
        const val STATUS_NO_MORE = 103
        const val STATUS_FAILURE = 104
    }
}
